local/getDevices.go

```diff
@@ -3,69 +3,222 @@ package local
 import (
 	"context"
 	"fmt"
+	"sort"
+	"sync"
 
-	"github.com/go-ocf/sdk/schema"
+	"github.com/go-ocf/kit/log"
 
-	gocoap "github.com/go-ocf/go-coap"
+	codecOcf "github.com/go-ocf/kit/codec/ocf"
+	kitStrings "github.com/go-ocf/kit/strings"
+	ocf "github.com/go-ocf/sdk/local/core"
+	ocfschema "github.com/go-ocf/sdk/schema"
```

Comment:
The import aliases `ocf` and `ocfschema` are redundant.

Reply:
no it's not duplicit import